scripts/inference.py

```diff
@@ -81,6 +82,18 @@
 )
 parser.add_argument("--context_file", type=str, default=None, help="File to summarize")
 
+parser.add_argument(
+    "--export_model",
+    action="store_true",
+    help="Export the compiled model using torch.export",
+)
+parser.add_argument(
```

Comment:
could export be satisfied by just the `export_path` argument? Do we need the `export_model` boolean as well?